config/routes.rb

```diff
@@ -42,7 +41,7 @@
   end
 
   scope controller: :users do
-    get "/dashboard"          => :show
+    get "/dashboard" => :show
```

Comment:
NITPICK: You can line them up like so:
`scope controller: :users do
    get "dashboard"              => :show 
    get "lookup_staff             => :lookup_staff_emails
    get "user_info/:user_id"  => :fetch_user_info
end`
Just makes it neat